### DISCUSSION TITLE
Date picker:  Incorrect Year Display in Date Picker with "th-TH" Locale after Losing Focus

### DISCUSSION THREAD

**Discussion by piyapanw** [chakra-ui/zag#2062] [General]
When using "th-TH" as the locale, after selecting a date and the textbox loses focus, the year is displayed incorrectly.

stackblitz: [https://stackblitz.com/edit/npqkaz?file=src%2FDatePicker.tsx](url)
screencast: [Screencast from 2024-10-02 22-40-31.webm](https://github.com/user-attachments/assets/3581d5ac-0dd8-45cd-bbe8-b862094e52bd)

**Comment by segunadebayo** [comment]
Moving this to a discussion for now, until we can collectively figure out a plan of attack
